test/coverage-test/test/threshold-glob.test.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'vitest'
+import { coverageTest, isV8Provider, normalizeURL, runVitest, test } from '../utils'
+import { sum } from '../fixtures/src/math'
+import { isEven, isOdd } from '../fixtures/src/even'
+
+test('threshold glob patterns count in global coverage', async () => {
+  await runVitest({
+    include: [normalizeURL(import.meta.url)],
+    coverage: {
+      all: false,
+      include: ['**/fixtures/src/**'],
+      thresholds: {
+        'branches': 100,
+        'functions': 50,
+        'lines': isV8Provider() ? 66 : 50,
+        'statements': isV8Provider() ? 66 : 50,
+
+        '**/fixtures/src/even.ts': {
+          branches: 100,
+          functions: 100,
+          lines: 100,
+          statements: 100,
+        },
+      },
```

Comment:
I've adjusted these now so that following error is shown when the fix is reverted.

```
FAIL  |v8| test/threshold-glob.test.ts > threshold glob patterns count in global coverage
Error: ERROR: Coverage for lines (50%) does not meet global threshold (66%)
ERROR: Coverage for functions (25%) does not meet global threshold (50%)
ERROR: Coverage for statements (50%) does not meet global threshold (66%)

FAIL  |istanbul| test/threshold-glob.test.ts > threshold glob patterns count in global coverage
Error: ERROR: Coverage for lines (25%) does not meet global threshold (50%)
ERROR: Coverage for functions (25%) does not meet global threshold (50%)
ERROR: Coverage for statements (25%) does not meet global threshold (50%)
```